panel/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@holoviz/panel",
-  "version": "0.9.4",
+  "version": "0.9.4-1",
```

Comment:
I would suggest using a full version string like `0.9.4-dev.1` or `0.9.4-rc.1` and publishing with an appropriate tag, i.e. `npm publish --access=public --tag=dev` (or `--tag=rc` respectively). For production release you can skip `--tag` altogether or use `--tag=latest` (that's the default).

Reply:
Also version should be kept in `package.json`, then `npm install` will update `package-lock.json`.

Reply:
So this was a build number generated using `npm version build` because I messed up the initial release of 0.9.4 and did `npm unpublish`, so I wouldn't ordinarily do this. I also realized too late that you can't republish after unpublishing, which means that the currently Python package won't be able to load the 0.9.4 release from the CDN :(

Will have to tag a 0.9.5 soon to fix it.

Reply:
> (...) can't republish after unpublishing (...)

This used to be possible on npm, but it's so unsafe that it was removed not that long ago.

Reply:
Yeah, it's fair enough, although I wish it gave you a 5 minute window or something or until the first person downloaded it.